frontend/src/App.jsx

```diff
@@ -1,8 +1,33 @@
-import React from "react";
-import Loader from "./Components/Loader/Loader";
+import React, { useState } from "react";
+import { Routes, Route } from "react-router-dom";
+import HomePage from "./Components/HomePage/HomePage";
+import Login from "./Components/Login/Login";
+import SignUp from "./Components/SignUp/SignUp";
+import ProtectedRoute from "./Components/Routes/ProtectedRoute";
+import RedirectRoute from "./Components/Routes/RedirectRoute";
 
 const App = () => {
-	return <Loader />;
+	const [accessToken, setAccessToken] = useState(localStorage.accessToken);
+
+	return (
+		<Routes>
+			<Route path="/" exact element={<RedirectRoute accessToken={accessToken} />} />
+			<Route path="login" exact element={<Login setAccessToken={setAccessToken} />} />
+			<Route path="signup" exact element={<SignUp />} />
+			<Route
+				path="homepage"
```

Comment:
why having homepage, why not homepage is just "/"? No big deal, just a bit cleaner.

Reply:
Great suggestion.
I'll fix it, thanks :)